.github/workflows/ci_tests.yml

```diff
@@ -174,6 +187,9 @@ jobs:
         run: cargo nextest archive --archive-file nextest-archive.tar.zst
 
       - name: Run Catalyst Core tests
+        env:
+          TEST_DB: postgres
+          TEST_DATABASE_URL: postgres://postgres:123456@localhost
```

Comment:
This can be done better later. It seems I named the test db env vars differently in different parts of vit-ss

Reply:
OK